app/src/main/res/values/dimens.xml

```diff
@@ -10,7 +10,6 @@
     <dimen name="textSize32">32sp</dimen>
     <dimen name="textSize36">36sp</dimen>
 
-    <dimen name="spacing0">0sp</dimen>
     <dimen name="spacing4">4sp</dimen>
```

Comment:
ja tego nie zauwazylem wczesniej ale przeczytaj jaka jest roznica miedzy `dp` a `sp` i popraw to